### PR TITLE
Languages were added.

### PR DESCRIPTION
**Pull request by patrycja93**
For now only English and Polish are supported.
Default for automated tests is English.

Changes in JudgeStatements were required due to added
functionality.

### CODE REVIEW

File: src/main/java/pl/patrycja/ox/ui/ConsoleUI.java [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
     public ConsoleUI(String[] args) {
         this.args = args;
         scanner = new Scanner(System.in);
+        inputChecker = new InputChecker(this);
         checkInputParameters();
     }
 
     private void checkInputParameters() {
-        if (!inputChecker.checkIfInputParametersAreValid(args)) {
+        if (inputChecker.checkIfInputParametersAreNotValid(args)) {
```
Comment (BulaMar) [inline src/main/java/pl/patrycja/ox/ui/ConsoleUI.java:26]: 👍

File: src/main/java/pl/patrycja/ox/ui/ConsoleUI.java [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
     private Scanner scanner;
-    private InputChecker inputChecker = new InputChecker();
+    private InputChecker inputChecker;
     private String[] args;
+    private ResourceBundle messages;
 
     public ConsoleUI(String[] args) {
         this.args = args;
         scanner = new Scanner(System.in);
+        inputChecker = new InputChecker(this);
```
Comment (BulaMar) [inline src/main/java/pl/patrycja/ox/ui/ConsoleUI.java]: This is circular dependency, think about which should have which.

File: src/main/java/pl/patrycja/ox/ui/Sequence.java [hunk 3]
```diff
@@ -18,7 +18,6 @@
         this.towards = Integer.parseInt(args[2]);
     }
 
-    //TODO: generate draw sequence!
```
Comment (BulaMar) [inline src/main/java/pl/patrycja/ox/ui/Sequence.java:21]: 👍

File: src/test/java/pl/patrycja/ox/TestUI.java [hunk 4]
```diff
@@ -4,11 +4,21 @@
 
 public class TestUI implements UI {
 
+    @Override
+    public void display(String input) {
+
```
Comment (BulaMar) [inline src/test/java/pl/patrycja/ox/TestUI.java:9]: also add comments why empty if need to be empty

File: src/test/java/pl/patrycja/ox/TestUI.java [hunk 5]
```diff
@@ -4,11 +4,21 @@
...
+    }
+
     @Override
     public void display(Object input) {
         //empty for tests purposes
     }
 
+    @Override
+    public void display(String message, Object... args) {
+
```
Comment (BulaMar) [inline src/test/java/pl/patrycja/ox/TestUI.java]: same

File: src/test/java/pl/patrycja/ox/TestUI.java [hunk 6]
```diff
@@ -19,4 +29,9 @@ public String read() {
     public int readNumber() {
         return 0;
     }
+
+    @Override
+    public void getLanguage() {
+
```
Comment (BulaMar) [inline src/test/java/pl/patrycja/ox/TestUI.java:37]: and here

File: src/main/java/pl/patrycja/ox/gameflow/AutomaticTests.java [hunk 7]
```diff
@@ -43,6 +43,7 @@
 
     @Override
     public void settings(String[] inputArrayParameters) {
+        ui.getLanguage();
```
Comment (BulaMar) [inline src/main/java/pl/patrycja/ox/gameflow/AutomaticTests.java:46]: maybe this can be in mode?

**Review by BulaMar — APPROVED** [review]
Small remarks but generally okey

File: src/main/java/pl/patrycja/ox/ui/ConsoleUI.java (hunk 2 above)
Comment (patrycja93) [reply src/main/java/pl/patrycja/ox/ui/ConsoleUI.java]: Yes, I will check it :)

**Pull request merged**